pkg/controller/account/account_controller.go

```diff
@@ -54,7 +54,7 @@ const (
 	caseDesiredInstanceLimit      = 25
 	caseStatusResolved            = "resolved"
 	intervalAfterCaseCreationSecs = 30
-	intervalBetweenChecksSecs     = 30
+	intervalBetweenChecksSecs     = 10
```

Comment:
The variable name has Secs, yet we use it as minutes. Can you rename it please?

Reply:
good catch